lib/mix/lib/mix/tasks/compile.elixir.ex

```diff
@@ -47,6 +50,14 @@ defmodule Mix.Tasks.Compile.Elixir do
   def run(args) do
     {opts, _, _} = OptionParser.parse(args, switches: @switches)
 
+    # The :long_compilation_threshold is in seconds, but we want to pass it to
+    # the compiler in milliseconds.
+    opts =
+      case Keyword.fetch(opts, :long_compilation_threshold) do
```

Comment:
@josevalim just us in tests so that we can pass like 50 ms and make it fast, but I guess we can make it seconds everywhere and use `0` if we need to test it.